### PR TITLE
Feature/nec 20/add lithuanian language

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/category.rdf.po [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+"Language: lt-LT\n"
+"sourceLanguage: en-US\n"
+"targetLanguage: lt-LT\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+# http://www.tao.lu/Ontologies/TAOItem.rdf#ExposeCategory
+msgctxt "http://www.w3.org/2000/01/rdf-schema#label"
+msgid "Expose to category auto-add"
+msgstr ""
```
Comment (tikhanovichA) [inline locales/lt-LT/category.rdf.po]: Is it should be not translatable?

Comment (AndreyNegrov) [reply locales/lt-LT/category.rdf.po]: it's normal

File: locales/lt-LT/messages.po [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+#, tao-public
+msgid "Preview"
+msgstr "Peržiūra"
+
+#, tao-public
+msgid "Properties"
+msgstr "Savybės"
+
+msgid "Published \"%1$s\" in language \"%2$s"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]:
```suggestion
msgid "Published \"%1$s\" in language \"%2$s\""
```

File: locales/lt-LT/messages.po [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+msgstr "Paskelbta \"%1$s\", kalba \"%2$s"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]:
```suggestion
msgstr "Paskelbta \"%1$s\", kalba \"%2$s\""
```

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

**Pull request merged**